### ISSUE TITLE
Add flags to readme

### ISSUE THREAD

**Issue by gdudek** [mholt/timeliner#23]
I suggest documenting the flags, and their usage, in the readme file.
They can be found in main.go or via "**timeliner -help**"

For example:

timeliner -repo /Volumes/XDrive get-all google_photos/me@gmail.com

**Comment by mholt** [comment]
Yeah, will do, after I solidify the CLI more. I might move most of the flags into a config file.
